src/ensembl/package.json

```diff
@@ -34,7 +34,7 @@
   "lint-staged": {
     "*.{ts,tsx}": [
       "eslint --max-warnings=0",
-      "npx manage-licence-header add",
+      "manage-licence-header add",
```

Comment:
Note to all reviewers: we found that the `npx` command was unnecessary, because after running `npm install`, the `manage-licence-header` was already in `node_modules/.bin` and can just be called as above.